zio/ndjsonio/config_test.go

```diff
@@ -1,9 +1,10 @@
-package ndjsonio
+package ndjsonio_test
```

Comment:
these ndjson changes looks unrelated to csv, are they needed?

Reply:
sorry, rebasing troubles.  will fix.